Traceback (most recent call last):
  File "O:\Envs\nycudopcs\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "O:\Envs\nycudopcs\Lib\site-packages\nbclient\client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "O:\Envs\nycudopcs\Lib\site-packages\jupyter_core\utils\__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\jaen8\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "O:\Envs\nycudopcs\Lib\site-packages\nbclient\client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "O:\Envs\nycudopcs\Lib\site-packages\nbclient\client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "O:\Envs\nycudopcs\Lib\site-packages\nbclient\client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Find the square root of a perfect square number
x = int(input("Please enter an integer: "))
# Start with a guess
g = 0
while g**2 < x:
    g += 1
# Check the result
if g**2 != x:
    print(x, "is not a perfect square number.")
else:
    print("The square root of {} is {}.".format(x, g))

------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
Cell [1;32mIn[1], line 2[0m
[0;32m      1[0m [38;5;66;03m# Find the square root of a perfect square number[39;00m
[1;32m----> 2[0m x [38;5;241m=[39m [38;5;28mint[39m([38;5;28;43minput[39;49m[43m([49m[38;5;124;43m"[39;49m[38;5;124;43mPlease enter an integer: [39;49m[38;5;124;43m"[39;49m[43m)[49m)
[0;32m      3[0m [38;5;66;03m# Start with a guess[39;00m
[0;32m      4[0m g [38;5;241m=[39m [38;5;241m0[39m

File [1;32mO:\Envs\nycudopcs\Lib\site-packages\ipykernel\kernelbase.py:1201[0m, in [0;36mKernel.raw_input[1;34m(self, prompt)[0m
[0;32m   1199[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m[38;5;241m.[39m_allow_stdin:
[0;32m   1200[0m     msg [38;5;241m=[39m [38;5;124m"[39m[38;5;124mraw_input was called, but this frontend does not support input requests.[39m[38;5;124m"[39m
[1;32m-> 1201[0m     [38;5;28;01mraise[39;00m StdinNotImplementedError(msg)
[0;32m   1202[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39m_input_request(
[0;32m   1203[0m     [38;5;28mstr[39m(prompt),
[0;32m   1204[0m     [38;5;28mself[39m[38;5;241m.[39m_parent_ident[[38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m],
[0;32m   1205[0m     [38;5;28mself[39m[38;5;241m.[39mget_parent([38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m),
[0;32m   1206[0m     password[38;5;241m=[39m[38;5;28;01mFalse[39;00m,
[0;32m   1207[0m )

[1;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.

